tikv/test_util.go

```diff
@@ -93,3 +94,41 @@ func NewTestTiKVStore(client Client, pdClient pd.Client, clientHijack func(Clien
 	tikvStore.mock = true
 	return tikvStore, err
 }
+
+// NewTestTiKVStore creates a test store with Option
+func NewTestKeyspaceTiKVStore(client Client, pdClient pd.Client, clientHijack func(Client) Client, pdClientHijack func(pd.Client) pd.Client, txnLocalLatches uint, keyspaceMeta keyspacepb.KeyspaceMeta, opt ...Option) (*KVStore, error) {
+	codec, err := apicodec.NewCodecV2(apicodec.ModeTxn, &keyspaceMeta)
+	if err != nil {
+		return nil, err
+	}
+	client = &CodecClient{
+		Client: client,
+		codec:  codec,
+	}
+
+	codecPDCli, err := locate.NewCodecPDClientWithKeyspace(apicodec.ModeTxn, pdClient, keyspaceMeta.Name)
+	if err != nil {
+		return nil, err
+	}
+	pdCli := pd.Client(codecPDCli)
+
+	if clientHijack != nil {
+		client = clientHijack(client)
+	}
+
+	if pdClientHijack != nil {
+		pdCli = pdClientHijack(pdCli)
+	}
+
+	// Make sure the uuid is unique.
+	uid := uuid.New().String()
+	spkv := NewMockSafePointKV()
```

Comment:
Should this spkv add a prefix too?

Reply:
It is a `map[string]string` ，not a etcd storage at here.
So we don't need the prefix, right?